recipe/meta.yaml

```diff
@@ -1,61 +1,283 @@
+      string: h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}{{ linkage_nature }}{{ debug }}_cpython
+      run_exports:
+        noarch:
+          - python
+        weak:
+          - python_abi 3.8.* *_cp38
+    # Not until: https://github.com/mingwandroid/conda-build/commit/c68a7d100866df7a3e9c0e3177fc7ef0ff76def9
+    # {% if 'conda-forge' not in channel_targets %}
+    #      - CONDA_FORGE=yes
+    # {% else %}
+    #      - CONDA_FORGE=no
+    # {% endif %}
+    script: build_base.sh  # [unix]
+    script: build_base.bat  # [win]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
+        - {{ cdt('libx11-devel') }}  # [linux]
+        - {{ cdt('libuuid-devel') }}  # [linux and not (ppc64le or aarch64)]
+        - make  # [not win]
+        - pkg-config  # [not win]
+        # configure script looks for llvm-ar for lto
+        - llvm-tools  # [osx]
+        - patch  # [not win]
+        - m2-patch  # [win]
+        - posix  # [win]
+        - ld_impl_{{ target_platform }} >=2.34  # [linux]
+        - binutils_impl_{{ target_platform }} >=2.34  # [linux]
+        - libtool  # [unix]
+      host:
+        - bzip2  # [not win]
+        - sqlite
+        - xz  # [not win]
+        - zlib  # [not win]
+        - openssl
+        - readline  # [not win]
+        - tk  # [not win]
+        - ncurses  # [linux]
+        - libffi  # [not win]
+        - ld_impl_{{ target_platform }} >=2.34  # [linux]
+      run:
+        - ld_impl_{{ target_platform }} >=2.34  # [linux]
+      run_constrained:
+        - python_abi 3.8.* *_cp38
+    test:
+      requires:
+        - ripgrep
+        - cmake
+        - ninja
+        # - cython
+        # - setuptools
+        - {{ compiler('c') }}
+        # Tried to use enable_language(C) to avoid needing this. It does not work.
+        - {{ compiler('cxx') }}
+      files:
+        - tests/distutils/*
+        # - tests/distutils.cext/*
+        - tests/cmake/*
+        - tests/cython/*
+        - tests/prefix-replacement/*
+      commands:
+        - echo on  # [win]
+        - set  # [win]
+        - python -V
+        - python3 -V            # [not win]
+        - 2to3 -h
+        - pydoc -h
+        - python3-config --help  # [not win]
+        - set "PIP_NO_BUILD_ISOLATION=False"  # [win]
+        - set "PIP_NO_DEPENDENCIES=True"  # [win]
+        - set "PIP_IGNORE_INSTALLED=True"  # [win]
+        - set "PIP_NO_INDEX=True"  # [win]
+        - set "PIP_CACHE_DIR=%CONDA_PREFIX%/pip_cache"  # [win]
+        - set "TEMP=%CONDA_PREFIX%/tmp"  # [win]
+        - mkdir "%TEMP%"  # [win]
+        - python -Im ensurepip --upgrade --default-pip  # [win]
+        - python -c "from zoneinfo import ZoneInfo; from datetime import datetime; datetime(2020, 10, 31, 12, tzinfo=ZoneInfo('America/Los_Angeles')); print(dt.tzname())"
```

Comment:
I expected this to fail only on Windows but it seems that our macOS and Linux images do not have the zoneinfo data. Question? Should we add `tzdata` to all platforms or should we add it only on Windows and do not perform this tests on the other platforms?

Most of the time users will have that install by their systems already and according to https://www.python.org/dev/peps/pep-0615 Python will be smart enough to find it and prioritize it. What do you think @isuruf and @mingwandroid?